cpu/stm32/Makefile.dep

```diff
@@ -4,10 +4,17 @@
 USEMODULE += periph stm32_clk stm32_vectors
 
 ifneq (,$(filter periph_usbdev,$(FEATURES_USED)))
-  # TODO: STM32F105xx and STM32F107xx also use synopsys_dwc2
+  # All STM32 families except for STM32F1, STM32F3 and STM32WG use synopsys_dwc2
   ifeq (,$(filter f1 f3 wb,$(CPU_FAM)))
     USEMODULE += usbdev_synopsys_dwc2
   endif
+  # In STM32F1 family STM32F105xx and STM32F107xx also use synopsys_dwc2
+  ifneq (,$(filter stm32f103%,$(CPU_FAM)))
+    USEMODULE += usbdev_synopsys_dwc2
+  endif
+  ifneq (,$(filter stm32f105%,$(CPU_FAM)))
+    USEMODULE += usbdev_synopsys_dwc2
+  endif
```

Comment:
```suggestion
ifneq (,$(filter stm32f105% stm32f107%,$(CPU_FAM)))
   USEMODULE += usbdev_synopsys_dwc2
endif
```
???

Reply:
STM32F103 has to use `periph/usbdev_fs` driver.

Reply:
Yes, it also has to be `CPU_MODEL` not `CPU_FAM` :/ Another CI round :cry:

Reply:
> Yes, it also has to be `CPU_MODEL` not `CPU_FAM` 

Oops, yes.

Reply:
looks like I'm getting erratic. But this time I think the fix is correct.